frontend/src/components/SideNav.tsx

```diff
@@ -257,302 +257,318 @@ export class SideNav extends React.Component<SideNavInternalProps, SideNavState>
     };
 
     return (
-      <div
-        id='sideNav'
-        className={classes(
-          css.root,
-          commonCss.flexColumn,
-          commonCss.noShrink,
-          collapsed && css.collapsedRoot,
-        )}
-      >
-        <div style={{ flexGrow: 1 }}>
-          {KFP_FLAGS.DEPLOYMENT === Deployments.MARKETPLACE && (
-            <>
+      <>
+        {KFP_FLAGS.DEPLOYMENT !== Deployments.KUBEFLOW && (
```

Comment:
Question: How do we make sure the central dashboard sideNav will show the same set of sideNav here? How do we make both pages keep in sync in the future?

Reply:
I will let @yanniszark articulate the current plan, we already had a discussion with the other members of the Notebooks WG, which covers Centraldashboard as well.

Reply:
That is great! Look forward to hear more about the plan.

Regarding this PR, should we first make the changes in Centraldashboard to implement these buttons, before submitting this PR for hiding buttons in KFP UI? Otherwise people will not able to see these buttons when they are in KUBEFLOW mode.

Reply:
@zijianjoy  See here https://github.com/kubeflow/pipelines/issues/5199#issuecomment-788697441 with that  we can move forward anyway, and the release is due in a couple of weeks anyway :)

Reply:
@zijianjoy this is a great question. As the release manager for 1.3, I have talked with members of the wg-notebooks team and have agreed on the following process:
1. Each Kubeflow WG decides on the version they want to include in manifests for the release. (this already happens)
2. The Notebooks WG changes the CentralDashboard ConfigMap (if needed) to contain entries for the decided version of each component. (this is how the ConfigMap will be updated)

In general, it would also be great for WGs to give a heads-up to the notebooks wg, or at least document changes in upgrade notes.

Reply:
Thank you @StefanoFioravanzo , sounds good with the HIDE_SIDENAV flag approach. And thank you @yanniszark for the ConfigMap upgrade process.